instance/tests/integration/integration_instance.py

```diff
@@ -379,13 +380,7 @@ def test_spawn_appserver(self):
 
         if settings.DEFAULT_INSTANCE_MYSQL_URL and settings.DEFAULT_INSTANCE_MONGO_URL:
             self.assert_swift_container_provisioned(instance)
```

Comment:
Is this check really needed now that swift support is deprecated? Can you remove it too?

Reply:
I intended to create a task to remove all deprecated code related to Swift in one go.

Given I'll have to get another CI run for the other change, I'll include removing this.